Automattic/wp-calypso
# Fix whitelisted coupon codes for Facebook ads.

#### Changes proposed in this Pull Request

* Change `0` in the word `0FF`, to `OFF` (`O`, not `0`)

#### Testing instructions

Test checkout by starting from calypso.live branch at `/start?coupon=FLASHFB50OFF`
- Confirm coupon is applied at checkout.
